generated/nidaqmx/_task_modules/channels/ai_channel.py

```diff
@@ -6733,7 +6752,7 @@ def ai_rng_low(self):
         check_for_error(error_code)
 
     @property
-    def ai_rosette_strain_gage_gage_orientation(self):
+    def ai_rosette_strain_gage_orientation(self):
```

Comment:
Same story here as eddy current. The old name was what LV uses (it has periods to break up the double gage). I like the new name, but lets add a deprecation.